README.md

```diff
@@ -48,3 +55,48 @@ bundle install
 bundle exec rake test
 bundle exec jekyll serve
 ```
+
+## Archiving the site
+
+We archive snapshots of the whole site for every release, which allows our customers to time travel and review
+older docs. This is very valuable if they're running older versions and need to look something up.
+Archives are maintained at https://archive.istio.io.
+
+To create an archive version, you need to follow these steps:
```

Comment:
I'm wondering if this is really something consumable by the typical istio contributor ?
should it go to https://github.com/istio/admin-sites/tree/master/archive.istio.io instead ?